### DISCUSSION TITLE
bibbot blocked by ublock

### DISCUSSION THREAD

**Discussion by mr0711** [stefanw/voebbot#426] [General]
Hello, i am just wondering why bibbot is not working. I found problems with ublock origin addon. Is there possibility to exclude bibbot from ublock? Which *.js have to be excluded?

Regards Patric

**Comment by stefanw** [comment]
On which page? It should not interfere with ublock because it runs as a content script and it's not loaded by the page.

**Reply by mr0711** [reply]
Site is https://www.handelsblatt.com

May there be some Problems with Option "Cookie notice" like it is describe here:

https://www.privacy-handbuch.de/screenshots/ublock-cookiebanner.png

**Reply by stefanw** [reply]
Generally speaking without testing this on a specific site: The add-on is trying to detect a paywall so it can do its thing. Paywalls are sometimes only loaded when you agree to some cookies. Possibly, ublock can prevent the cookie permission dialog or the paywall to appear which may prevent the add-on to detect the paywall.

**Reply by mr0711** [reply]
seems so

**Reply by ElbPirat** [reply]
You could check with the ublock network log, if there is something blocked and if so, what list blocked the script.
Network log:
![grafik](https://github.com/stefanw/bibbot/assets/88144925/358fe64c-9d2d-4606-8431-2366a75850db)

Reload your site and check log:
![grafik](https://github.com/stefanw/bibbot/assets/88144925/ba0ce1e8-9f6e-422f-9ae1-26f0044b4854)
Pubmatic get blocked by Easylist.
Scroll the list and check the red rows.
